docs/business.yaml

```diff
@@ -105,6 +105,49 @@ paths:
               schema:
                 type: object
                 properties: {}
+              examples:
+                response-consent-continuation-out:
+                  summary: Consent continuation out filing response
+                  value:
+                    filing:
+                      business:
+                        legalName: 0882848 B.C. LTD.
+                        legalType: BEN
+                        identifier: BC0882848
+                        foundingDate: '2024-07-08T15:34:57.844764+00:00'
+                      consentContinuationOut:
+                        foreignJurisdiction:
+                          country: CA
+                          region: SK
+                        courtOrder:
+                          fileNumber: '12345'
+                          effectOfOrder: planOfArrangement
+                      header:
+                        affectedFilings: []
+                        availableOnPaperOnly: false
+                        certifiedBy: "string"
+                        colinIds: []
+                        comments: []
+                        date: '2024-07-18'
+                        deletionLocked: false
+                        documentOptionalEmail: "user@example.com"
+                        effectiveDate: '2024-07-18T07:00:42.552434+00:00'
+                        email: "user@example.com"
+                        filingId: 150056
+                        inColinOnly: false
+                        isCorrected: false
+                        isCorrectionPending: false
+                        isPaymentActionRequired: true
+                        name: consentContinuationOut
+                        paymentAccount: '3445'
+                        paymentStatusCode: 'CREATED'
+                        paymentToken: '38416'
+                        status: 'PENDING'
+                        submitter: 'api-key-account-3445-sandbox'
```

Comment:
Only cover 2 error responses now.
Plan to update with additional error responses for other potential scenarios.